package.json

```diff
@@ -4,31 +4,34 @@
   "private": true,
   "scripts": {
     "android": "react-native run-android",
-    "ios": "react-native run-ios",
+    "ios": "react-native run-ios --simulator=\"iPhone SE (3rd generation)\"",
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint .",
     "start:appium": "appium",
-    "test:e2e": "jest --testMatch=\"**/__tests__/**/*E2E.test.js\"",
+    "test:e2e": "jest --testMatch=\"**/__tests__/**/*E2E.test.tsx\"",
     "test:e2e:all": "yarn test:e2e:android && yarn test:e2e:ios",
     "test:e2e:android": "E2E_DEVICE=\"android\" yarn test:e2e",
     "test:e2e:ios": "E2E_DEVICE=\"ios\" yarn test:e2e"
   },
   "dependencies": {
+    "@types/react-native": "^0.71.1",
     "react": "18.0.0",
     "react-native": "0.69.4"
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
     "@babel/runtime": "^7.12.5",
     "@react-native-community/eslint-config": "^2.0.0",
-    "appium": "^1.22.3",
+    "@types/jest": "^29.4.0",
+    "appium": "^2.0.0-beta.53",
+    "appium-xcuitest-driver": "^4.16.12",
     "babel-jest": "^26.6.3",
     "eslint": "^7.32.0",
     "jest": "^26.6.3",
-    "webdriverio": "^7.21.0",
     "metro-react-native-babel-preset": "^0.70.3",
-    "react-test-renderer": "18.0.0"
+    "react-test-renderer": "18.0.0",
+    "webdriverio": "^7.30.1"
```

Comment:
To be updated in another PR

Reply:
[branch](https://github.com/MadeinFrance/react-native-e2e-jest-appium-webdriverio/commit/d0faecfa1c115605dd649d128f3475081a9fda57#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R43)

Issue:
```
node_modules/webdriverio/build/index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import logger from '@wdio/logger';

SyntaxError: Cannot use import statement outside a module
```